charts/mattermost-enterprise-edition/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Enterprise server with high availitibity.
 name: mattermost-enterprise-edition
-version: 1.20.0
+version: 1.21.0
```

```suggestion
version: 1.20.1
```

just notice, this is just a small fix there is no features